src/compute-client/src/controller/replicated.rs

```diff
@@ -642,17 +718,13 @@ where
         let replica_state = ReplicaState {
             command_tx,
             response_rx,
-            _task: task.abort_on_drop(),
```

Comment:
What is the reason to remove this? Seems like a good safe-guard to make sure we always clean up the task, even if we through some bug fail to do it otherwise.

Reply:
Good point!